Tests/test_image_access.py

```diff
@@ -301,17 +302,19 @@ def test_embeddable(self) -> None:
     return 0;
 }}
         """
-            )
+                )
 
-        objects = compiler.compile(["embed_pil.c"])
-        compiler.link_executable(objects, "embed_pil")
+            objects = compiler.compile(["embed_pil.c"])
+            compiler.link_executable(objects, "embed_pil")
 
-        env = os.environ.copy()
-        env["PATH"] = sys.prefix + ";" + env["PATH"]
+            env = os.environ.copy()
+            env["PATH"] = sys.prefix + ";" + env["PATH"]
 
-        # Do not display the Windows Error Reporting dialog
-        getattr(ctypes, "windll").kernel32.SetErrorMode(0x0002)
+            # Do not display the Windows Error Reporting dialog
+            getattr(ctypes, "windll").kernel32.SetErrorMode(0x0002)
 
-        process = subprocess.Popen(["embed_pil.exe"], env=env)
-        process.communicate()
-        assert process.returncode == 0
+            process = subprocess.Popen(["embed_pil.exe"], env=env)
+            process.communicate()
+            assert process.returncode == 0
+        finally:
+            os.remove("embed_pil.c")
```

Comment:
Shall we add `teardown_method` to this class to delete it, to save an indent?

https://docs.pytest.org/en/stable/how-to/xunit_setup.html#method-and-function-level-setup-teardown

Reply:
Ok, I've pushed a commit.